lua/gitlinker/routers.lua

```diff
@@ -130,6 +137,13 @@ local function bitbucket_blame(lk)
   return builder:build("annotate")
 end
 
+--- @param lk gitlinker.Linker
+--- @return string
+local function codeberg_blame(lk)
+  local builder = Builder:new(lk, lines_range)
```

Comment:
`lines_range` => `LC_range`

Reply:
BTW, if you could add two more test case, that will be perfect:
1. test the string template with [gitlinker._browse](https://github.com/linrongbin16/gitlinker.nvim/blob/9d3f8ded388dee2e1d49be424609b66a9db77bf9/test/gitlinker_spec.lua#L74) and [gitlinker._blame](https://github.com/linrongbin16/gitlinker.nvim/blob/9d3f8ded388dee2e1d49be424609b66a9db77bf9/test/gitlinker_spec.lua#L92).
2. test the api function with [routers.codeberg_browse](https://github.com/linrongbin16/gitlinker.nvim/blob/9d3f8ded388dee2e1d49be424609b66a9db77bf9/test/routers_spec.lua#L22) and [routers.codeberg_blame](https://github.com/linrongbin16/gitlinker.nvim/blob/9d3f8ded388dee2e1d49be424609b66a9db77bf9/test/routers_spec.lua#L58).